source/developers-guide/plugin-quick-start/index.md

```diff
@@ -525,3 +595,520 @@ Possible settings for the parameter `caches` contains each context as constant.
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/plugin.xsd">
+    <label lang="de">Slogan des Tages</label>
+    <label lang="en">Slogan of the day</label>
+
+    <version>1.0.0</version>
+    <link>http://example.org</link>
+    <author>shopware AG</author>
+    <compatibility minVersion="5.2.0" />
+
+    <changelog version="1.0.0">
+        <changes lang="de">Farbe geändert; Schriftgröße geändert;</changes>
+        <changes lang="en">changed color; changed font-size;</changes>
+    </changelog>
+</plugin>
+```
+
+## Require other plugins
+If you need other plugins to be installed, then you are able to define them in your plugin.xml file like this
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/plugin.xsd">
+    ...
+    <changelog version="1.0.0">
+        ...
+    </changelog>
+    <requiredPlugins>
+        <requiredPlugin pluginName="SwagRequiredPlugin" minVersion="1.2.0" maxVersion="3.5.2"/>
+    </requiredPlugins>
+</plugin>
+```
+
+## Plugin Configuration / Forms
+
+Backend plugin configuration can be extended by `Resources/config.xml` file. This replaces the usage of `$this->Form()` on old `Shopware_Components_Plugin_Bootstrap`.
+
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/config.xsd">
+    <elements>
+        <element required="true" type="text" scope="locale">
+            <name>slogan</name>
+            <label lang="de">Dein Slogan</label>
+            <label lang="en">Your slogan</label>
+            <value>XML is fun!</value>
+            <description>This slogan will be shown on the storefront</description>
+        </element>
+    </elements>
+</config>
+```
+
+To read out the configuration of your plugin use this code snippet in your base plugin class:
+
+```php
+$config = $this->container->get('shopware.plugin.cached_config_reader')->getByPluginName($this->getName());
+```
+Use it like this in other places:
+```php
+$config = Shopware()->Container()->get('shopware.plugin.cached_config_reader')->getByPluginName('SwagSloganOfTheDay');
+```
+The config reader service will return an array with the config element names as keys. If you are using scoped configuration elements, don't forget to pass the shop service `$this->container->get('Shop')` or `Shopware()->Shop() as a second parameter to `getByPluginName()` method. Without the shop context, the method will fall back to the main shop context.
+
+### add store values
+As of Shopware 5.2.11 it is possible to define custom config stores directly inside your `config.xml`.
```

Comment:
did we say we use Shopware 5.3.0 as min version for the whole docs? so all the hints for the version are useless, aren't they?

please also check other places

Reply:
d